linting/pylintrc

```diff
@@ -11,7 +11,8 @@
 #    C0326: Bad whitespace
 # 5. fixme (left 'TODO' lines)
```

Comment:
Is this file supposed to have the prefix `.` , ie: `.pylintrc`?

Reply:
The [idea was](https://github.com/populationgenomics/team-docs/blob/main/python_code_style.md#setting-up-a-new-project) that you download those with `wget` when setting up a new project. And I wasn't sure how the URLs would look with `.` prefixes. Perhaps and overprotection, I'll try prefixing back with `.`

Reply:
Ah yep, too easy :)

Reply:
I'll actually keep them without `.` to reflect that those are examples to grab, not actual rc files. Also with `.`, the terminal hides them, and a user can miss that there are more files in the folder. Doesn't help to `cp *` them all as `cp` omits the hidden files by default.